app/src/main/java/com/infomaniak/mail/workers/DraftsActionsWorker.kt

```diff
@@ -203,6 +204,7 @@ class DraftsActionsWorker(appContext: Context, params: WorkerParameters) : BaseC
                         it.messageUid = data?.messageUid
                         it.action = null
                     }
+                    scheduledDate = dateFormatWithTimezone.format(Date())
```

Comment:
Why did this fix the issue ? Didn't sending null amount to the same ?

Reply:
Yes, sending `null` amounts to the same result, but the code will be more complicated to understand.